tests/UI/campaigns/productV2/functional/18_seoTab.ts

```diff
@@ -0,0 +1,273 @@
+// Import utils
+import helper from '@utils/helpers';
+import testContext from '@utils/testContext';
+
+// Import common tests
+import loginCommon from '@commonTests/BO/loginBO';
+
+// Import BO pages
+import dashboardPage from '@pages/BO/dashboard';
+import createProductPage from '@pages/BO/catalog/productsV2/add';
+import seoTab from '@pages/BO/catalog/productsV2/add/seoTab';
+import productsPage from '@pages/BO/catalog/productsV2';
+
+// Import FO pages
+import foProductPage from '@pages/FO/product';
+import {searchResultsPage} from '@pages/FO/searchResults';
+import categoryPageFO from '@pages/FO/category';
+
+// Import data
+import ProductData from '@data/faker/product';
+import Products from '@data/demo/products';
```

Comment:
```suggestion
import Products from '@data/demo/products';
import ProductData from '@data/faker/product';
```

Reply:
What's the problem ?

Reply:
The order ;) 

We put import by alphabetic order in general.